src/WebView.windows.tsx

```diff
@@ -67,34 +69,38 @@ export default class WebView extends React.Component<WebViewSharedProps, State>
 
   webViewRef = React.createRef<NativeWebViewWindows>();
 
+  RnwVersionSupportsWebView2 = (version.major>1 || version.minor>=68);
+  
+  RCTWebViewString = (this.RnwVersionSupportsWebView2 && this.props.useWebView2) ? 'RCTWebView2' : 'RCTWebView';
```

Comment:
Could be possible the (react native used) version is 1.68+ is running but the system native `RCTWebView2` module is not available in the system?
(could be an outdated system)
Is there any reliable way of checking if the `RCTWebView2` is available?

Reply:
how about windows versions that are not compatible with webview2? such as the `Window team` device family? (the one used for Surface Hubs)

Reply:
If the runtime is not installed, the app will show a message where the webview would be, telling you to click on this link to install the runtime. 

The problem is afaik Surface Hub can only install stuff from the store, and the webview2 runtime is not available in the store. 
I'm not on the webview2 team, but if you have feedback/questions about it, the right issue to raise them is MicrosoftEdge/WebView2Feedback#1851